File: Beautiful Days at the Movies/beautiful-days-at-the-movies.py
# Complete the beautifulDays function below.
def beautifulDays(i: 'int', j: 'int', k: 'int') -> 'int':
    """
    :type i: int
    :type j: int
    :type k: int
    :rtype: int
    """
    count = 0

    while i <= j:
        count += abs(i - int(str(i)[::-1])) % k == 0
        i += 1

    return count


print(beautifulDays(20, 23, 6))  # 2
print(beautifulDays(13, 45, 3))  # 33

File: Beautiful Days at the Movies/beautiful-days-at-the-movies.js
/**
 * Complete the beautifulDays function in the editor below. It must return the number of beautiful days in the range.
 *
 * @return {number}
 * @param {number} i
 * @param {number} j
 * @param {number} k
 */
const beautifulDays = (i, j, k) => {
  let count = 0;

  const check = (v) => {
    return Math.abs(v - parseInt(v.toString().split("").reverse().join(""))) % k === 0;
  };

  while (i <= j) {
    count += check(i);
    i++;
  }

  return count;
};

console.log(beautifulDays(20, 23, 6)); // 2
console.log(beautifulDays(13, 45, 3)); // 33